### PR TITLE
Replace "API" by "Integration" key for PagerDuty

### PR DESCRIPTION
**Pull request by annelaurefroment**
**What this PR does / why we need it**:
In the instructions for setting up a PagerDuty Alert Notification there is a reference to an API key in the text but an Integration key in the table.  They are different in PagerDuty, it should be "Integration" in both places.  
It would also be helpful to also reference that "Integration" keys are per "Service" in PagerDuty so people know where to look for them.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/alerting/notifications.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ If you are using the token for a slack bot, then you have to invite the bot to t
 
 ### PagerDuty
 
-To set up PagerDuty, all you have to do is to provide an API key.
+To set up PagerDuty, all you have to do is to provide an Integration key.
```
Comment (aknuds1) [inline docs/sources/alerting/notifications.md]:
```suggestion
To set up PagerDuty, all you have to do is to provide an integration key.
```

**Review by aknuds1 — DISMISSED** [review]

**Review by marefr — APPROVED** [review]
LGTM, but agree there's no need to capitalize Integration.

**Review by ryantxu — APPROVED** [review]
LGTM

**Comment by ryantxu** [comment]
@annelaurefroment -- thank you for the contribution, can you sign the CLA so we can merge your change?  thanks

**Review by aknuds1 — APPROVED** [review]
LGTM

**Pull request merged**